test/e2e/tokens.wdio.conf.js

```diff
@@ -0,0 +1,296 @@
+require('@okta/env'); // Read environment variables from "testenv"
+require('@babel/register'); // Allows use of import module syntax
+require('regenerator-runtime'); // Allows use of async/await
+
+const DEBUG = process.env.DEBUG;
+const CI = process.env.CI;
+const defaultTimeoutInterval = DEBUG ? (24 * 60 * 60 * 1000) : 10000;
+const logLevel = CI ? 'warn' : 'info';
+const browserOptions = {
+    args: []
+};
+
+if (CI) {
+    browserOptions.args = browserOptions.args.concat([
+        '--headless',
+        '--disable-gpu',
+        '--window-size=1600x1200',
+        '--no-sandbox',
+        '--whitelisted-ips',
+        '--disable-extensions',
+        '--verbose'
+    ]);
+}
+
+exports.config = {
+    jasmineNodeOpts: {
+        defaultTimeoutInterval,
+        stopSpecOnExpectationFailure: true
+    },
+
+
+    //
+    // ====================
+    // Runner Configuration
+    // ====================
+    //
+    // WebdriverIO allows it to run your tests in arbitrary locations (e.g. locally or
+    // on a remote machine).
+    runner: 'local',
+    //
+    // Override default path ('/wd/hub') for chromedriver service.
+    path: '/',
+    //
+    // ==================
+    // Specify Test Files
+    // ==================
+    // Define which test specs should run. The pattern is relative to the directory
+    // from which `wdio` was called. Notice that, if you are calling `wdio` from an
+    // NPM script (see https://docs.npmjs.com/cli/run-script) then the current working
+    // directory is where your package.json resides, so `wdio` will be called from there.
+    //
+    specs: [
+        './specs/**/refreshTokens.js', './specs/**/crossTabs.js'
```

Comment:
If the only config diff is here, how about use the env var (REFRESH_TOKEN) to build the specs array? so only one wdio config needs to be maintained.